src/Microsoft.Azure.ServiceBus/ServiceBusConnection.cs

```diff
@@ -309,7 +310,8 @@ TransportSettings CreateTransportSettings()
                 return AmqpConnectionHelper.CreateWebSocketTransportSettings(
                     networkHost: networkHost,
                     hostName: hostName,
-                    port: port);
+                    port: port,
+                    proxy: WebRequest.DefaultWebProxy);
```

Comment:
It seems weird that this is required. ClientWebSocketOptions should ideally just use the defaultWebProxy. 
Have you been able to get a repro of proxy working after this change, and also made sure it doesn't work on your setup without this change?
In the full framework, it already uses the defaultWebProxy and hence it has been working.

Reply:
Yes, it does not work behind the proxy without this change running on .NET Core 2.1 on Windows 10. When running in the full Framework it works without this change.

Reply:
That is interesting. Thanks